draw/reddit.dart

```diff
@@ -0,0 +1,9 @@
+// Provides the reddit class
+
+class reddit{
```

Comment:
Done, following was the output for dart analyser:
$ Analyzing [reddit.dart]...
$ No issues found